### DISCUSSION TITLE
Can Nightingale alerts call out to other systems?

### DISCUSSION THREAD

**Discussion by loveyang2012** [ccfos/nightingale#1164] [Ideas]
nightingale的告警系统能否开放给其他系统调用，通过自定义模板，把告警发出去？

**Comment by UlricQin** [comment]
不支持。我估计你的需求是其他系统产生告警事件，推到一个统一的地方，这个统一的地方支持多种渠道的发送，甚至支持聚合降噪之类的，这个我们后面会有一个专门的产品来解决这个场景，是个SaaS产品
